extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -397,6 +419,19 @@ export class AmpStoryPage extends AMP.BaseElement {
     return this.isPrerenderActivePage_;
   }
 
+  /**
+   * Returns true if the page is the last amp-story-page in the amp-story.
+   * @return {boolean}
+   */
+  isLastPage() {
```

Comment:
Nit: `@private` and a trailing underscore `isLastPage_`

Reply:
Done. I had to change the name of the private variable (and I wasn't using it correctly for caching the value anyway).